### PR TITLE
DLP: Added sample for Inspect Datastore Job with SCC integration

### PR DESCRIPTION
**Pull request by jainamtrivedi-crest**
- Added sample for [Inspect Datastore Job With SCC Integration](https://cloud.google.com/dlp/docs/sending-results-to-scc.md?g=0&l=416#datastore-data) and unit test cases

### CODE REVIEW

File: dlp/api/Snippets/InspectDataStoreJobWithSCCIntegration.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+        // Specify the type of info to be inspected and construct the inspect config.
+        var inspectConfig = new InspectConfig
+        {
+            InfoTypes =
+            {
+                infoTypes ?? new InfoType[]
+                {
+                    new InfoType { Name = "EMAIL_ADDRESS" },
+                    new InfoType { Name = "PERSON_NAME" }
```
Comment (annadata) [inline dlp/api/Snippets/InspectDataStoreJobWithSCCIntegration.cs:59]: Please align info types with [Java sample](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/fafc6dca9dad437b9de9062d8f54168396fb0109/dlp/snippets/src/main/java/dlp/snippets/InspectDatastoreSendToScc.java#L84).

File: dlp/api/Snippets.Tests/InspectDataStoreJobWithSCCIntegrationTests.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+{
+    public class InspectDataStoreJobWithSCCIntegrationTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public InspectDataStoreJobWithSCCIntegrationTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestCreate()
+        {
+            var result = InspectDataStoreJobWithSCCIntegration.SendDataStoreData(_fixture.ProjectId, "kindId", "namespaceId");
```
Comment (annadata) [inline dlp/api/Snippets.Tests/InspectDataStoreJobWithSCCIntegrationTests.cs:27]: Does this sample need resources to be created?

Comment (jainamtrivedi-crest) [reply dlp/api/Snippets.Tests/InspectDataStoreJobWithSCCIntegrationTests.cs:27]: No Bhaskar. It does not require any resources to be created. And it does not throw any error while running the sample or test case itself.

File: dlp/api/Snippets/InspectDataStoreJobWithSCCIntegration.cs (hunk 1 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/InspectDataStoreJobWithSCCIntegration.cs]: Done.

File: dlp/api/Snippets/InspectDataStoreJobWithSCCIntegration.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+using System.Collections.Generic;
+using System.Linq;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using static Google.Cloud.Dlp.V2.InspectConfig.Types;
+
+public class InspectDataStoreJobWithSCCIntegration
+{
+    public static DlpJob SendDataStoreData(
```
Comment (annadata) [inline dlp/api/Snippets/InspectDataStoreJobWithSCCIntegration.cs:25]: Please rename method name. How about: SendInspectDatastoreToSCC

File: dlp/api/Snippets.Tests/InspectDataStoreJobWithSCCIntegrationTests.cs (hunk 2 above)
Comment (annadata) [reply dlp/api/Snippets.Tests/InspectDataStoreJobWithSCCIntegrationTests.cs:27]: Please rename "kindId" to "your-datastore-kindId" and "your-namespace-id"
Effectively this only confirms that the job request is well formed.

**Review by annadata — APPROVED** [review]

File: dlp/api/Snippets.Tests/InspectDataStoreJobWithSCCIntegrationTests.cs (hunk 2 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets.Tests/InspectDataStoreJobWithSCCIntegrationTests.cs:27]: Updated as per your suggestion. Thanks.

File: dlp/api/Snippets/InspectDataStoreJobWithSCCIntegration.cs (hunk 3 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/InspectDataStoreJobWithSCCIntegration.cs:25]: Updated. Thanks

**Review by amanda-tarafa — APPROVED** [review]

**Pull request merged**